source/php/Modules/FrontendForm/FormAdmin.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace EventManager\Modules\FrontendForm;
+
+use WpService\Contracts\AddFilter;
+use WpService\Contracts\GetPostTypeObject;
+use WpService\Contracts\GetPostType;
+use AcfService\Contracts\GetFieldGroups;
+
+class FormAdmin
+{
+    public function __construct(
+        private AddFilter&GetPostTypeObject&GetPostType $wpService,
+        private GetFieldGroups $acfService,
+        private string $fieldKey
+    ) {
+      $this->wpService->addFilter('acf/load_field/name=' . $fieldKey, [$this, 'addOptionsToGroupSelect']);
```

Comment:
Preffer implementing `Hookable` to avoid unexpected side effects in the constructor.

Reply:
Resolved by implementing hookable IF.